skill.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Laugh",
-    "url": "https://github.com/OpenVoiceOS/ovos-skill-laugh",
+    "url": "https://github.com/mikejgray/skill-laugh",
```

Comment:
This doesn't match the PR repo

Reply:
Correct, because it ran on my fork, but as soon as it merges this will be corrected automatically

Reply:
:facepalm: not sure why I read this as being in `setup.py`/package data. makes complete sense now